### PR TITLE
Provide demo build.rs

### PR DESCRIPTION
**Pull request by jamesmunns**
Consider using a `build.rs` instead of a `Makefile`. Not sure how to handle the log level info in your run.

Feel free to discard. I've just been using build scripts lately, thought it might be useful.

### CODE REVIEW

**Comment by jamesmunns** [comment]
The CI is failing because the CI server probably doesn't have `sass` installed. Its not a problem for `master` because you have a prebuilt css file in your repo.

**Comment by jamesmunns** [comment]
@heartsucker notice me senpai

**Comment by heartsucker** [comment]
@jamesmunns I tell `make` to ignore `sass` errors via `|| echo`. That's your problem. I also don't want to deal with managing gems on travis. I did that before in SecureDrop and it was a fucking nightmare. So updates to that will just get checked in and the Makefile is for me to be lazy and type slightly less.

File: build.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        .arg("--style")
+            .arg("expanded")
+        .arg("--sourcemap=none")
+        .arg("scss/styles.scss:static/css/styles.css")
+        .status()
+        .expect("scss not compiled :(")
+        .success());
+
+    // Only rebuild `build.rs` process if these these have changed:
+    println!("cargo:rerun-if-changed=build.rs");
```
Comment (heartsucker) [inline build.rs:19]: Do these prints have an effect on `cargo`'s behavior? If so, docs?

Comment (jamesmunns) [reply build.rs:19]: @heartsucker sorry for the delay, here are the docs: http://doc.crates.io/build-script.html#outputs-of-the-build-script

**Pull request closed without merging**